<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
    <line>
      <![CDATA[Test passed]]>
    </line>
  </reporter-output>
  <suite name="Bank" duration-ms="18998" started-at="2020-05-30T06:15:35Z" finished-at="2020-05-30T06:15:54Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="18998" started-at="2020-05-30T06:15:35Z" finished-at="2020-05-30T06:15:54Z">
      <class name="com.w2a.testcases.WindowHandleTest">
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:com.w2a.testcases.WindowHandleTest@3aeaafa6]" name="beforeTest" is-config="true" duration-ms="4070" started-at="2020-05-30T06:15:35Z" finished-at="2020-05-30T06:15:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="FAIL" signature="f(java.lang.String)[pri:0, instance:com.w2a.testcases.WindowHandleTest@3aeaafa6]" name="f" duration-ms="14902" started-at="2020-05-30T06:15:39Z" finished-at="2020-05-30T06:15:54Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.w2a.base.TestBase.takescreenshot(TestBase.java:43)
at com.w2a.listeners.Listenersclass.onTestSuccess(Listenersclass.java:41)
at org.testng.internal.TestListenerHelper.runTestListeners(TestListenerHelper.java:70)
at org.testng.internal.Invoker.runTestListeners(Invoker.java:1389)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:636)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- f -->
        <test-method status="PASS" signature="afterTest()[pri:0, instance:com.w2a.testcases.WindowHandleTest@3aeaafa6]" name="afterTest" is-config="true" duration-ms="5" started-at="2020-05-30T06:15:54Z" finished-at="2020-05-30T06:15:54Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- com.w2a.testcases.WindowHandleTest -->
    </test> <!-- Test -->
  </suite> <!-- Bank -->
</testng-results>
